stable_plugins/classical_ml/data_preparation/pandas_preprocessing/schemas.py

```diff
@@ -0,0 +1,271 @@
+# Copyright 2023 QHAna plugin runner contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from marshmallow import post_load, validate
+import marshmallow as ma
+from qhana_plugin_runner.api.util import (
+    FrontendFormBaseSchema,
+    MaBaseSchema,
+    FileUrl,
+)
+from qhana_plugin_runner.api import EnumField
+
+from dataclasses import dataclass
+from .backend.pandas_preprocessing import (
+    PreprocessingEnum,
+    AxisEnum,
+    KeepEnum,
+    PositionEnum,
+    CaseEnum,
+)
+
+from celery.utils.log import get_task_logger
+
+
+class TaskResponseSchema(MaBaseSchema):
+    name = ma.fields.String(required=True, allow_none=False, dump_only=True)
+    task_id = ma.fields.String(required=True, allow_none=False, dump_only=True)
+    task_result_url = ma.fields.Url(required=True, allow_none=False, dump_only=True)
+
+
+@dataclass(repr=False)
+class FirstInputParameters:
+    file_url: str
+
+    def __str__(self):
+        return str(self.__dict__)
+
+
+class FirstInputParametersSchema(FrontendFormBaseSchema):
+    file_url = FileUrl(
+        required=True,
+        allow_none=False,
+        metadata={
+            "label": "File URL",
+            "description": "A url to the file.",
```

Comment:
Remove redundant descriptions if the label already communicates the same information (the description just uses screenspace without providing any benefits in this case)